pb/api/v1/data.proto

```diff
@@ -162,6 +167,85 @@ message ListRowsResponse {
   }
 }
 
+message GetDashboardTabRequest {
+  string scope = 1;
+  string dashboard = 2;
+  string tab = 3;
+}
+
+message GetDashboardTabResponse {
+
+  // GCSConfig specifies results stored in GCS, typically created by prow.
+  message GCSConfig {
+    // Path to the test result stored in gcs (some-bucket/some/optional/path).
+    string gcs_prefix = 1;
+  
+    // The GCP project where GCS sends notifications for the above prefix.
+    string pubsub_project = 2;
```

Comment:
I think we can remove the pubsub fields for now, they're easy to add later if needed.